index.html

```diff
@@ -0,0 +1,156 @@
+               <li class="box-1">DC Ignite</li>
+               <p class="box-2">Join our webinars and live training sessions.</p>
+            </div>
+          </div>
+        </ul>
+        <a class="mp-1">SEE THE WHOLE PROGRAM</a>
+        <button class="button">Join Interior Symposium 2022</button>
+     </div>
+    </section>
+
+      <section class=".section-speakers-heading container-fluid featured-speakers text-dark bg-light.bg-gradient">
+        <div class="row pt-3 mb-5">
+          <div class="col-12">
+            <h3 class="featured">Featured Speakers</h3>
+            <hr class="hr-title">
+          </div>
+        </div>
+          <div class="col-12 text-center mb-5 border d-block d-md-none">
+            <a data-toggle="collapse" data-target="#demo" href="#" class="text-uppercase btn rounded-0 pr-5 pl-5 pt-2 pb-2">
+              More <i class="fas fa-angle-down red"></i>
+            </a>
+          </div>
+      </section> 
```

Comment:
- [ ] Kindly make sure that the pages  look almost identical to the original design. For Mobile screen size In the feature speaker section design look different from the original design. Please try to adjust it.
Your Project Look likes this : 
![C1](https://user-images.githubusercontent.com/93598189/191455764-da7061ec-105a-470f-9f7f-591beaf20897.PNG)

The Template design Look Like this : 

![C2](https://user-images.githubusercontent.com/93598189/191455904-796fb336-6cd8-4bf5-8bac-8cdaf055f2bd.PNG)